tests/test_tpt_reports.py

```diff
@@ -135,3 +135,12 @@ def test_domain_validation():
         except SystemExit as sys_exit:
             return_code = sys_exit.code
             assert return_code == 2, "main() should return with error return code 2"
+
+
+def test_load_json_file():
+    """Validate file loads correctly."""
+    assert tpt_reports.tpt_reports.load_json_file("./tests/data/test.json") is not None
```

Comment:
As mentioned in https://github.com/cisagov/tpt-reports/pull/52#discussion_r1337213879, it may make sense to make `tests/data/test.json` a global constant.

Related, you should standardize on how file paths are referenced, either `"./tests/data/test.json"` (as you do in this PR) or `"tests/data/test.json"` (as is being done in https://github.com/cisagov/tpt-reports/pull/52).

Reply:
Resolved in https://github.com/cisagov/tpt-reports/pull/51/commits/5540dbaf322459d9435da0d930eed20e588143a1